HowTo/use_go_with_appservice.md

```diff
@@ -0,0 +1,112 @@
+# Deploy a Go web app to Azure App Service
+
+In this quickstart, you'll deploy a Go web app to Azure App Service. Azure App Service is a fully managed web hosting service that supports Go 1.18 and higher apps hosted in a Linux server environment.
+
+To complete this quickstart, you need:
+
+1. An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs).
+2. [Go 1.18](https://go.dev/dl/) or higher installed locally.
+
+## 1 - Sample Application
+
+First, create a folder for your project.
+
+Go to the terminal window, change into the folder you just created and run `go mod init <ModuleName>`. The ModuleName could just be the folder name at this point.
+
+The `go mod init` command creates a go.mod file to track your code's dependencies. So far, the file includes only the name of your module and the Go version your code supports. But as you add dependencies, the go.mod file will list the versions your code depends on.
+
+Create a file called main.go. We will be doing most of our coding here.
+
+```golang
+
+package main
+
+import (
+    "fmt"
+    "net/http"
+)
+
+func main() {
+    http.HandleFunc("/", HelloServer)
+    http.ListenAndServe(":8080", nil)
+}
+
+func HelloServer(w http.ResponseWriter, r *http.Request) {
+    fmt.Fprintf(w, "Hello, %s!", r.URL.Path[1:])
+}
+
+```
+
+This program use the `net.http` package to handle all requests to the web root with the HelloServer function. The call to `http.ListenAndServe` tells the server to listen on the TCP network address :8080
+
+If you want to do a sanity check, you can run this by opening up a terminal, navigating to your project’s directory and then calling `go run main.go`. Now open a browser window and type the URL `http://localhost:8080/world`. You should see the message `Hello, world!`.
+
+## 2 - Create a web app in Azure
+
+To host your application in Azure, you need to create Azure App Service web app in Azure. You can create a web app using the Azure CLI.
+
+Azure CLI commands can be run on a computer with the [Azure CLI installed](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+Azure CLI has a command `az webapp up` that will create the necessary resources and deploy your application in a single step.
+
+If necessary, login to Azure using [az login](https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli).
+
+```azurecli
+az login
+```
+
+Create the webapp and other resources, then deploy your code to Azure using [az webapp up](https://learn.microsoft.com/en-us/cli/azure/webapp#az-webapp-up).
+
+```azurecli
+az webapp up --runtime GO:1.18 --sku B1 --logs
```

Comment:
`az webapp up --runtime GO:1.18 --sku B1 --logs`

Got this error : Windows runtime 'GO|1.18' is not supported. Please check supported runtimes with: 'az webapp list-runtimes --os Windows'

this worked `az webapp up --runtime GO:1.18 --sku B1 --os Linux --logs`